<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_choose_image">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gpy">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>View Gaussian Pyramids</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lpy">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>View Laplacian Pyramids</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Seam Carving with Image Pyramids</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Rows : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Cols   : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_rows_del">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_rows_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Rows to delete   : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_cols_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Cols to delete     : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_cols_del">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_rows">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_cols">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_proceed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_rem">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save/Remove Objects</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
